stix_shifter_modules/secretserver/stix_transmission/api_client.py

```diff
@@ -124,15 +129,22 @@ def decode_searchId(self):
                 3001, "Could not decode search id content - " + self.search_id)
         self.query = jObj.get("query", None)
         try:
-            timestamp = re.findall(r'\d{4}-\d{2}-\d{2}', self.query)
+            pattern = re.compile("\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{3}Z")
+            matcher = pattern.findall(self.query)
+            matches = []
+            for match in matcher:
+                dt = parser.parse(match)
+                matches.append(dt.strftime("%Y-%m-%d %H:%M:%S"))
         except:
             raise IOError(
                 " Could not extract date- " + self.search_id)
-        return timestamp
+        # return timestamp
+        return matches
 
     def get_events(self):
         payload = "{\"name\": \"Secret Server Events Logs\", \"parameters\": [{\"name\": \"startDate\", \"value\": '%s'} , {\"name\":\"endDate\",\"value\": '%s'}]}" % (
             self.startDate, self.endDate)
+        # payload = "{\n     \"name\":\"Secret Server Events Logs\" ,\n    \n     \"parameters\": [\n        {\n           \"name\": \"startDate\",\n            \"value\": \"2021-09-13 05:22:37.617\"\n        } ,\n        {\n            \"name\": \"endDate\",\n            \"value\": \"2021-09-13 19:50:49.293\"\n        }\n\n      \n    ]\n    \n}\n"
```

Comment:
I see there are some commented out codes. you should remove them if you don't need.